### PR TITLE
FP-322/Implement New Preset Configs

### PR DESCRIPTION
**Pull request by Kevoniatom**

### CODE REVIEW

File: template/template.json [hunk 1]
```diff
@@ -81,9 +83,30 @@
...
+    },
+    {
+      "name": "Vuetify",
+      "plugins": [
+        "vuetify",
+        "localization",
+        "pwa",
+        "fontawesome"
+      ],
+      "descrition": "Vuetify, Localization, FontAwesome, PWA"
```
Comment (OrandiH) [inline template/template.json:105]: *description

File: template/template.json [hunk 2]
```diff
@@ -81,9 +83,30 @@
...
+  "presets": [
+    {
+      "name": "Recommended",
+      "plugins": [
+        "buefy",
+        "localization",
+        "pwa",
+        "fontawesome"
+      ],
+      "descrition": "Buefy, Localization, FontAwesome, PWA"
```
Comment (OrandiH) [inline template/template.json:95]: *description

**Review by OrandiH — CHANGES_REQUESTED** [review]
@Kevoniatom see comments.

**Comment by Kevoniatom** [comment]
updated

**Comment by Kevoniatom** [comment]
@OrandiH

**Review by OrandiH — APPROVED** [review]
@Kevoniatom approved and ready for merge.

**Review by OrandiH — CHANGES_REQUESTED** [review]
Changes needed.

**Review by OrandiH — APPROVED** [review]

**Pull request merged**